### DISCUSSION TITLE
How to clear benchmarks

### DISCUSSION THREAD

**Discussion by skyguynca** [RainbowMiner/RainbowMiner#1671] [Q&A] [answered]
I updated my cpu's and need to run new benchmarks, ubuntu server 20.04. How can I do that without and re-installation.

Thanks
David

**Comment by kuroshio** [comment, marked as answer]
In the browser on the Benchmarks page (http://localhost:4000/minerstats.html) you can click Benchmark All Miners:

![Screen Shot 2021-07-18 at 2 32 12 AM](https://user-images.githubusercontent.com/61211787/126058005-aeeb31b7-555d-4b74-8ebe-aacccbf2b4e9.png)
